### PR TITLE
Permission

### PR DESCRIPTION
**Pull request by zmbq**
Version 0.5. Only thing left is update the docker-compose file so we run the official 0.5.0 backend.

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,3 +1,3 @@
 VUE_APP_MODE = development
-VUE_APP_BACKEND_PREFIX = http://localhost:5000
+VUE_APP_BACKEND_PREFIX = http://localhost:8080
```
Comment (Bronson-Brown-deVost) [inline .env.development:2]: shouldn't this still be port 5000?

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
 
     sqe-http:
         container_name: SQE_HTTP_API
-        image: qumranica/sqe-http-api:0.4.5-pre6
+        image: qumranica/sqe-http-api:0.5.0-pre12
```
Comment (Bronson-Brown-deVost) [inline docker/docker-compose.yml:25]: If you want, you can change this now to 0.5.0, as you mentioned at the top of the PR.

File: src/components/navigation/Navbar.vue [hunk 3]
```diff
@@ -111,6 +111,7 @@ export default class Navbar extends Vue {
 
     private logout() {
         this.sessionService.logout();
+        router.push('/');
```
Comment (Bronson-Brown-deVost) [inline src/components/navigation/Navbar.vue:114]: Nice!

File: src/i18n/en/navbar.ts [hunk 4]
```diff
@@ -26,5 +26,8 @@ export default {
     changeForgottenPassword: 'Change Forgotten Password',
 
     activateUser: 'Activate User',
-    activate: 'Activate'
+    activate: 'Activate',
+
+    confirmInvitation: 'Comfirm Invitation',
```
Comment (Bronson-Brown-deVost) [inline src/i18n/en/navbar.ts:31]: should be Confirm not Comfirm (sorry for the nitpick).

File: src/views/edition/components/confirm-invitation.vue [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        </form>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import ErrorService from '@/services/error';
+import router from '@/router';
+import {
+    ResetForgottenUserPasswordRequestDTO,
```
Comment (Bronson-Brown-deVost) [inline src/views/edition/components/confirm-invitation.vue:33]: You probably don't need this do you?

File: .env.development (hunk 1 above)
Comment (zmbq) [reply .env.development:2]: Oh, that's a remnant of the CORS issue. That's how we overcame it. I'm not sure whether this is an actual issue or a misconfiguration. We need to test this on another environment.

**Pull request merged**